docs/js/native.js

```diff
@@ -1,101 +1,102 @@
 'use strict';
-var _native = {
+const _native = {
```

Comment:
Can we please undo the changes to this file, and ignore this file in eslint? This file was provided by BSA, I don't want to risk changing it.

Reply:
sure, though i dont think anything practically changed

it will be ignored like `.min.js` files

Reply:
reverted & ignored with eb349911cb25d9987ae15c82df3378ea9d9b5752